librz/core/cmd/cmd_eval.c

```diff
@@ -53,8 +53,41 @@ static bool pal_seek(RzCore *core, RzConsPalSeekMode mode, const char *file, RzL
 	return true;
 }
 
-RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
+static bool print_meta_info(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *MetaInfo, const char *name) {
 	bool failed = false;
+
+	if (MetaInfo->author != NULL) {
+		rz_cons_printf("%s", rz_str_append(MetaInfo->author, '\0'));
+	}
+	if (MetaInfo->license != NULL) {
+		rz_cons_printf("%s", rz_str_append(MetaInfo->license, '\0'));
+	}
+	if (MetaInfo->targetbg != NULL) {
+		rz_cons_printf("%s", rz_str_append(MetaInfo->targetbg, '\0'));
+	}
+	if (MetaInfo->mincolordepth != NULL) {
+		rz_cons_printf("%s", rz_str_append(MetaInfo->mincolordepth, '\0'));
```

Comment:
why this rz_cons_printf in this way?

Reply:
Was not sure if the copied strings were terminated by a null char, and wanted to avoid segfaults, hence did this null char appending😅